src/Observable.ts

```diff
@@ -211,7 +211,7 @@ export class Observable<T> implements CoreOperators<T>  {
                projectResult?: (x: T, y: any, ix: number, iy: number) => R,
                concurrent?: number) => Observable<R>;
   flatMapTo: <R>(observable: Observable<any>, projectResult?: (x: T, y: any, ix: number, iy: number) => R, concurrent?: number) => Observable<R>;
-  groupBy: <K, R>(keySelector: (value: T) => string,
+  groupBy: <T, K, R>(keySelector: (value: T) => K,
```

Comment:
do we need to add `T` as function generic parameter? expect `T` comes from generic type of `Observable<T>`.